hack/log/redact.sh

```diff
@@ -45,9 +45,9 @@ for log_file in "${log_files[@]}"; do
         # LC_CTYPE=C and LANG=C will prevent "illegal byte sequence" error from sed
         if [[ "$(uname)" == "Darwin" ]]; then
             # sed on Mac OS requires an empty string for -i flag
-            LC_CTYPE=C LANG=C sed -i "" "s|${redact_var}|===REDACTED===|g" "${log_file}" || true
+            LC_CTYPE=C LANG=C sed -i "" "s|${redact_var}|===REDACTED===|g" "${log_file}" > /dev/null || true
```

Comment:
You might need &> /dev/null, I still see a bunch of `sed: -e expression #1, char 0: no previous regular expression` in the logs